.github/2.0/cookbooks/Document.md

```diff
@@ -564,10 +567,30 @@ from jina import Document
 q = Document()
 # get match Document `m`
 m = Document()
-m.score.value = 0.96
+m.score = 0.96
 q.matches.append(m)
 ```
 
+```text
+<jina.types.document.Document id=184049a9-cac1-11eb-8d87-e86a64801cb1 scores=[{'value': 0.96, 'op_name': 'score', 'ref_id': '184049a8-cac1-11eb-8d87-e86a64801cb1'}] adjacency=1 at 140519623882704>
+```
+
+You can also set `score` or `evaluation` with a `metric` name by calling:
+```python
+from jina import Document
+
+d = Document()
+d.set_score('cosine distance', 0.5)
+d.set_score('relevance', 10.0)
```

Comment:
why `cosine similarity` is represented as a score, not a field in tag?

Reply:
`score` is a `special` field in `Document`